docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>i think we can find something nicer. I'll try to put the code for figure 4 in our <a href="https://arxiv.org/abs/2302.04065" target="_blank">paper</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>To me there is a big tradeoff to find in terms of simplicity of the experiment and of the plotting : I'd really avoid having 15 lines of matplotlib code that does something prettier but less clear to the beginner. I have in mind the sklearn examples which use very rudimentary matplotlib (https://scikit-learn.org/stable/auto_examples/cluster/plot_bisect_kmeans.html#sphx-glr-auto-examples-cluster-plot-bisect-kmeans-py) but like that are cristal clear.</p>

Reply:
<p>With so many colors selected randomly (not ideal for color blind people) and points scattered all around (markers are the same for x / y), I have trouble visualizing anything in that plot. I agree that the code is more compact (definitely a +) but I feel we're losing in terms of visualization. Maybe something more intermediate?</p>